brion/plugin/compartmentReportHDF5.cpp

```diff
@@ -89,6 +98,38 @@ bool _verifyFile(const HighFive::File& file)
     return true;
 }
 
+size_t _parseCacheSizeOption(const URI& uri)
+{
+    const auto i = uri.findQuery("cache_size");
+    if (i == uri.queryEnd())
+        return 0;
+
+    const auto& value = i->second;
+    if (value == "auto")
+        return std::numeric_limits<size_t>::max();
```

Comment:
don't know too much about the intrinsics here, but this is more like 'max' cache size. Shall it not be depend on the size of the files or number of rows/frames/cells? 'auto' would imply something 'smart' to me.

Reply:
It's something smart, but I found a bit annoying to document it in the description. Auto means allocating space for a whole frame or trace, whichever is bigger. It's not really max, std::numeric_limits<size_t>::max() is used as a magic value to identify this request in the client code. To make it more semantic I've replaced it with a constant.